### DISCUSSION TITLE
Disable word wrap or make terminal columns configurable

### DISCUSSION THREAD

**Discussion by marky421** [isontheline/pro.webssh.net#1078] [Ideas]
Hello, I love your app and was very happy to buy it!

I eagerly await ProxyJump support...

In the meantime, one issue I have is that when I log into my servers, I have a custom motd that displays system stats via neofetch. When I open a session via webssh my motd shows up, but because it exceeds the column length for the webssh terminal, many of the lines wrap and it makes the whole thing look really messy and unreadable.

It would be great if you could implement a way to either disable the wraping behavior or make the column length configurable!

Thank you for your consideration!

**Comment by marky421** [comment]
I did find a workaround... I found that I can pinch to zoom out to reduce the font size and that does help. Thankfully the zoomed-out setting persists after closing the app. For now this works, but I would prefer to be able to scroll horizontally to accomodate more columns. This is helpful when veiwing tablular output in the terminal, such as when running htop.

**Reply by isontheline** [reply]
Hello @marky421 👋

Thank you so much for considering WebSSH and for your purchase 🙏
Word Wrap is needed because there is no horizontal scrollbar on the VT100 terminal emulator.

Have you some example of apps allowing to do horizontal scrolling?

**Reply by marky421** [reply]
Thank you for the response! No worries if it isn't possible due to limitations of the emulator. The app I used previously, Prompt 2 by Panic, may have supported horizontal scrolling (I honestly don't remember) but I do remember not having a problem seeing my motd without it needing to wrap. Perhaps it just rendered more columns? Unfortunately I don't have it installed anymore and the only version available now on the app store is the latest Prompt 3. I did find some old release notes for Prompt 2 that suggest it was possible to scroll horizontally: 

[https://help.panic.com/releasenotes/prompt2/](https://help.panic.com/releasenotes/prompt2/)

> Fixed an issue where scrolling left in htop caused header redraw issues.

**Reply by isontheline** [reply]
Thanks @marky421 for sharing about Prompt 🙏
I will take a look at it
